test/functional/pdo_sqlsrv/pdo_574_next_rowset.phpt

```diff
@@ -0,0 +1,154 @@
+--TEST--
+GitHub issue 574 - Fetch Next Result Test
+--DESCRIPTION--
+Verifies the functionality of PDOStatement nextRowset
+--ENV--
+PHPT_EXEC=true
+--SKIPIF--
+<?php require('skipif_mid-refactor.inc'); ?>
+--FILE--
+<?php
+
+require_once("MsCommon_mid-refactor.inc");
+
+try {
+    $conn = connect();
+    $tableName = 'test574';
+    $tableName1 = 'test574_1';
+
+    // create two tables with max fields
+    $columns = array(new ColumnMeta('varchar(max)', 'col1'));
+    createTable($conn, $tableName, $columns);
+
+    $columns = array(new ColumnMeta('varchar(max)', 'col1'));
+    createTable($conn, $tableName1, $columns);
+
+    // insert one row to each table
+    $phrase = str_repeat('This is a test ', 250);
+    $stmt = insertRow($conn, $tableName, array('col1' => $phrase));
+    unset($stmt);
+
+    $phrase = str_repeat('This is indeed very long ', 300);
+    $stmt = insertRow($conn, $tableName1, array('col1' => $phrase));
+    unset($stmt);
+
+    // run queries in a batch
+    $stmt = $conn->prepare("SELECT * FROM [$tableName]; SELECT artist FROM [cd_info]; SELECT * FROM [$tableName1]");
+    $stmt->execute();
+    
+    // fetch from $tableName
+    $row = $stmt->fetch(PDO::FETCH_NUM);
+    if ($row) {
+        echo(substr($row[0], 0, 15)) . PHP_EOL;
```

Comment:
this only echoes the first 10 characters, what if something has gone wrong in between? Would be great of you can compare it with `$phrase` instead